src/Microsoft.DotNet.ProjectJsonMigration/Rules/MigrateBuildOptionsRule.cs

```diff
@@ -114,11 +114,22 @@ private AddPropertyTransform<CommonCompilerOptions>[] KeyFileTransforms
             "**/*.cs"
         };
 
+        private static bool IsPlainFileName(string fileName)
+        {
+            return !fileName.Contains('/') && !fileName.Contains('\\');
```

Comment:
@jgoshi but then what about cases like `foo/../../*.cs`? If we are going to do that I'd rather write glob parser at this point unless we ONLY check for `/../ \..\ and startswith(../) and startswith(..\)`